sklearn/datasets/_samples_generator.py

```diff
@@ -609,6 +609,21 @@ def make_regression(
     coef : ndarray of shape (n_features,) or (n_features, n_targets)
         The coefficient of the underlying linear model. It is returned only if
         coef is True.
+
+    Examples
+    --------
+    >>> from sklearn.datasets import make_regression
+    >>> X, y = make_regression(
+    ...     n_samples=5, n_features=2, n_informative=0, noise=1, random_state=42
+    ... )
```

Comment:
n_informative=0 is a rather unusual setting. These docstring examples usually shows the basic usage of the function.